marathon/src/main/scala/io/buoyant/marathon/v2/Api.scala

```diff
@@ -27,8 +27,8 @@ object Api {
 
   case class UnexpectedResponse(rsp: http.Response) extends Throwable
 
-  def apply(client: Client, uriPrefix: String): Api =
-    new AppIdApi(client, s"$uriPrefix/$versionString")
+  def apply(client: Client, uriPrefix: String, marathonHealth: Boolean): Api =
```

Comment:
`marathonHealth` makes me think that this variable is an indication of marathon health.  Maybe rename to `checkMarathonHealth` or `useMarathonHealth` or something?